public/scripts/embedded.js

```diff
@@ -1,41 +1,72 @@
-var embeddedURI = 'http://localhost:3000/?isFromEmbedd=true&uri=http://4genderjustice.org/&primesource=archiveit&ci=1068&hd=4';//this usually has to be read dynamically, getting cors origin error here , to be resolved
-$(document).ready(function(){
+function embedImageGrid() {
+    $("#embedGrid").css("display", "block");
 
-	if(getParamValue("isFromEmbedd") == "true"){
+    var imageGridEmbed = $("#imageGrid .grid-container").html();
+    var regex = /<button\b[^>]*>(.*?)<\/button>/g;
+    imageGridEmbed = imageGridEmbed.replace(regex, "");
+    imageGridEmbed += `<link href=`+window.location.origin+`/styles/sitestyle.css" rel="stylesheet"/>
+    <style>
+        body{
+            padding-bottom: 0px;
+        }
+        ul.rig li {
+            padding-bottom: 20px;
+        }
+        ul.rig li img {
+            margin: 15px auto 20px auto;
+            display: block;
+        }
+        .gridimagedatetime {
+            font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+        }
+    </style>`;
 
-		$('.argumentsForm #urirIP').val(getParamValue("uri"));
-        $('.argumentsForm #collectionNo').val(getParamValue("ci"));
-        $('.argumentsForm #hammingDistance').val(getParamValue("hd"));
-        $('.argumentsForm input[value='+getParamValue("primesource") +']').prop("checked");
-		$(".getJSONFromServer").trigger('click');
-	}
+    var iframe = document.createElement('iframe');
+    iframe.src = 'data:text/html;charset=utf-8,' + encodeURIComponent(imageGridEmbed);
+    iframe.width = "100%";
+    iframe.height = "25%";
+    iframe.frameBorder = "0";
 
-});
-
-function triggerGo(){
-	if(getParamValue("isFromEmbedd") == "true"){
-
-		$('.argumentsForm #urirIP').val(getParamValue("uri"));
-        $('.argumentsForm #collectionNo').val(getParamValue("ci"));
-        $('.argumentsForm #hammingDistance').val(getParamValue("hd"));
-        $('.argumentsForm input[value='+getParamValue("primesource") +']').prop("checked");
-		$(".getJSONFromServer").trigger('click');
-	}
+    $("#embedGrid").val(new XMLSerializer().serializeToString(iframe));
+    $("#embedGrid").select();
 }
 
+function embedImageSlider() {
+    $("#embedSlider").css("display", "block");
 
-function getParamValue(paramName)
-{
-     var url = embeddedURI.split("?")[1]; //get rid of "?" in querystring
-    //var url = top.document.getElementById("tmvis_iframe").src.split("?")[1]; //get rid of "?" in querystring
-    var qArray = url.split('&'); //get key-value pairs
-    for (var i = 0; i < qArray.length; i++)
-    {
-        var pArr = qArray[i].split('='); //split key and value
-        if (pArr[0] == paramName)
-            return pArr[1]; //return value
+    var imageSliderEmbed = $("#imageSlider .imageslider_wrapper").html();
+    var regex = /<h2>(.*?)<\/h2>/gm;
+    imageSliderEmbed = imageSliderEmbed.replace(regex,"")
+    imageSliderEmbed += `<link href=`+window.location.origin+`/styles/sitestyle.css" rel="stylesheet"/>
```

Comment:
Why did you remove the opening double quote?

```suggestion
    imageSliderEmbed += `<link href="`+window.location.origin+`/styles/sitestyle.css" rel="stylesheet"/>
```

Reply:
Oops! Didn't realize that. I'll add them back. Thanks!